openapi/trello/client.bal

```diff
@@ -183,93 +183,91 @@ public isolated client class Client {
+    # + members - True or false 
+    # + memberFields - All or a comma-separated list of: avatarHash, bio, bioData, confirmed, fullName, idPremOrgsAdmin, initials, memberType, products, status, url or username 
+    # + checkItemStates - True or false 
+    # + checklists - One of: all or none 
+    # + 'limit - A number from 1 to 1000 
+    # + since - A date, or null 
+    # + before - A date, or null 
+    # + filter - One of: all, closed, none, open or visible 
+    # + fields - All or a comma-separated list of: badges, checkItemStates, closed, dateLastActivity, desc, descData, due, email, idAttachmentCover, idBoard, idChecklists, idLabels, idList, idMembers, idMembersVoted, idShort, labels, manualCoverAttachment, name, pos, shortLink, shortUrl, subscribed or url 
+    # + return - Success 
     remote isolated function getBoardsCardsByIdBoard(string idBoard, string? actions = (), string? attachments = (), string attachmentFields = "all", string? stickers = (), string? members = (), string memberFields = "avatarHash, fullName, initials and username", string? checkItemStates = (), string checklists = "none", string? 'limit = (), string? since = (), string? before = (), string filter = "visible", string fields = "all") returns http:Response|error {
         string  path = string `/boards/${idBoard}/cards`;
-        map<anydata> queryParam = {"actions": actions, "attachments": attachments, "attachment_fields": attachmentFields, "stickers": stickers, "members": members, "member_fields": memberFields, "checkItemStates": checkItemStates, "checklists": checklists, "limit": 'limit, "since": since, "before": before, "filter": filter, "fields": fields, 'key: self.apiKeys["key"], token: self.apiKeys["token"]};
+        map<anydata> queryParam = {"actions": actions, "attachments": attachments, "attachment_fields": attachmentFields, "stickers": stickers, "members": members, "member_fields": memberFields, "checkItemStates": checkItemStates, "checklists": checklists, "limit": 'limit, "since": since, "before": before, "filter": filter, "fields": fields, "key": self.apiKeyConfig.key, "token": self.apiKeyConfig.token};
         path = path + check getPathForQueryParam(queryParam);
         http:Response response = check self.clientEp-> get(path, targetType = http:Response);
         return response;
     }
     # Get boards cards by ID board by filter
     #
-    # + idBoard - Board_id
-    # + filter - Filter
-    # + return - Success
+    # + idBoard - Board_id 
+    # + filter - Filter 
+    # + return - Success 
     remote isolated function getBoardsCardsByIdBoardByFilter(string idBoard, string filter) returns http:Response|error {
         string  path = string `/boards/${idBoard}/cards/${filter}`;
-        map<anydata> queryParam = {'key: self.apiKeys["key"], token: self.apiKeys["token"]};
-        path = path + check getPathForQueryParam(queryParam);
         http:Response response = check self.clientEp-> get(path, targetType = http:Response);
```

Comment:
API Key is missing in this operation only. Please confirm whether it is not needed or not added mistakenly.

Reply:
Added to  root level